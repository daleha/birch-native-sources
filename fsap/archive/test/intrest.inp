PSDRR1.gen
g
2
temp.intrest.out
4
EcoRI
GAATTC
1
S
AccI
GTMKAC
2
S
BspMI
ACCTGC
10
14
q
0



