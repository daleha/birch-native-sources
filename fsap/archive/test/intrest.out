INTREST   Version  8/14/2001
PSDRR1  Configuration:  LINEAR  Length:          734 bp
                                           # of
                                  Cut     Sites Sites   Frags   Begin     End
EcoRI     GAATTC                   1          0
                                                          734       1     734

AccI      GTMKAC                   2          2
                                                  575     574       1     574
                                                  686     111     575     685
                                                           49     686     734

BspMI     ACCTGC                  10 ( 14)    1
                                                  119     616     119     734
                                                          118       1     118

